assets/js/googlesitekit/data/utils.js

```diff
@@ -187,6 +187,25 @@ export const collectSelectors = collect;
  */
 export const collectState = collect;
 
+/**
+ * Collects all store names.
+ *
+ * This function's main purpose is to ensure generated store names for a single store match.
+ *
+ * @since n.e.x.t
+ *
+ * @param {...string} args A list of store names, all of which must be equal.
+ * @return {string} The single store name.
+ */
+export const collectName = ( ...args ) => {
```

Comment:
This doesn't so much collect the names as verify them. It doesn't seem to be collecting any data–just verifying.

Maybe we want to change it to `checkStoreNames` or `ensureStoreNamesMatch`?

Reply:
I thought about that, but in a way it still collects the names and gives you the end result. Technically it's indeed more verifying than collecting, but I think this is more intuitive in combination with the other functions. As a developer, you basically just have to remember "for every piece of the store I need to use `collect{typeOfPiece}`. So I'd prefer to keep this name.

If it was a more standalone-intended utility function, I would agree that it should be something more about verifying.

Reply:
Fair enough, that point about not having to remember a different name for this is a good one. 👍